### PR TITLE
Challenge 2

### PR DESCRIPTION
**Pull request by beatriz-cortes-94**
Adding tests for home view

### CODE REVIEW

File: src/components/YoutubeGrid/YoutubeGrid.test.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Grid from './YoutubeGrid.component';
+
+describe('card', () => {
```
Comment (rpadro) [inline src/components/YoutubeGrid/YoutubeGrid.test.jsx:5]: The description name should be related to the component you are testing

File: src/components/YoutubeNav/YoutubeNav.test.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Nav from './YoutubeNav.component';
+
+describe('Nav elements', () => {
+  test('must display navbar', () => {
+    render(<Nav />);
+    const icons = screen.getAllByRole('navigation');
+    expect(icons).toHaveLength(1);
```
Comment (rpadro) [inline src/components/YoutubeNav/YoutubeNav.test.jsx:9]: I think `toBeInTheDocument` could be a more appropriate assertion for this.

Comment (rpadro) [reply src/components/YoutubeNav/YoutubeNav.test.jsx:9]: Unless there are more than one icon with the role navigation, and I would use getByRole without the `all`.

**Review by rpadro — COMMENTED** [review]
Can you add a screenshot with the coverage or save results into a .txt file at ./ - root level.

**Comment by beatriz-cortes-94** [comment]
I Will address these comments in challenge_3 pr

**Pull request merged**